src/terminal/parser/tracing.cpp

```diff
@@ -103,19 +98,15 @@ void ParserTracing::DispatchSequenceTrace(const bool fSuccess)
 
 void ParserTracing::ClearSequenceTrace()
 {
-    ZeroMemory(_rgwchSequenceTrace, sizeof(_rgwchSequenceTrace));
-    _cchSequenceTrace = 0;
+    _sequenceTrace.clear();
 }
 
 // NOTE: I'm expecting this to not be null terminated
-void ParserTracing::DispatchPrintRunTrace(const wchar_t* const pwsString, const size_t cchString) const
+void ParserTracing::DispatchPrintRunTrace(const std::wstring_view string) const
 {
-    size_t charsRemaining = cchString;
-    wchar_t str[BYTE_MAX + 4 + sizeof(wchar_t) + sizeof('\0')];
-
-    if (cchString == 1)
+    if (string.size() == 1)
     {
-        wchar_t wch = *pwsString;
+        wchar_t wch = *string.cbegin();

```

Comment:
this is actually string.at[0] or string.front() eh?

Reply:
front in this case, imo.